executor/executor_fuchsia.h

```diff
@@ -22,6 +22,15 @@ static intptr_t execute_syscall(const call_t* c, intptr_t a[kMaxArgs])
 {
 	intptr_t res = c->call(a[0], a[1], a[2], a[3], a[4], a[5], a[6], a[7], a[8]);
 	if (strncmp(c->name, "zx_", 3) == 0) {
+		// Convert zircon error convention to the libc convention that executor expects.
+		// The following calls return arbitrary integers instead of error codes.
+		if (res == ZX_OK ||
```

Comment:
Seems like this is the only condition we actually need to keep, but if you want to revert for now and clean up later that seems fine too.

Reply:
These syscalls returning random values like `zx_clock_get` should now be marked with `ignore_return` attribute instead of hard-coding here. Similar to how 4ce69996ec362f8dd9762dcc1643d13cebaab44a did for some syscalls.
This is required for so-called "fallback coverage". If kernel code coverage is not available syzkaller will use syscall error codes as some notion of fuzzing feedabck. Random values returned by zx_clock_get pollute corpus w/o benefit.

Reply:
Ok! We will fix that logic soon to re-revert the commit (just leave the logic for ZX_OK = 0).

I want to first land all the other PRs so I unblock the rest of the team